### PR TITLE
Add manifest.json to Zuperkulblog

### PR DESCRIPTION
**Pull request by frankiefu**

### CODE REVIEW

File: templates/publishing/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    },{
+      "src": "//app-layout-assets.appspot.com/assets/zuperkulblog/launcher-icon-384x384.png",
+      "sizes": "384x384",
+      "type": "image/png"
+    }],
+  "start_url": "index.html",
+  "background_color": "#fafafa",
+  "display": "standalone",
+  "theme_color": "#fafafa"
+}
```
Comment (blasten) [inline templates/publishing/manifest.json]: needs a line. otherwise it LGTM

Comment (frankiefu) [inline templates/publishing/manifest.json]: :+1:

Comment (keanulee) [inline templates/publishing/manifest.json]: newline?

Comment (blasten) [inline templates/publishing/manifest.json]: yeah at the end of the file

Comment (frankiefu) [inline templates/publishing/manifest.json]: done

**Pull request merged**